composer.json

```diff
@@ -49,10 +49,11 @@
         "psr/http-message": "1.0.*",
         "ramsey/uuid": "~3",
         "google/proto-client": "^0.24",
-        "google/gax": "^0.24"
+        "google/gax": "^0.24",
+        "paragonie/random_compat": "^1|^2"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.*",
+        "phpunit/phpunit": "^4.8|^5.0",
```

Comment:
Allowing composer to choose between versions gives us better error reporting on PHP 7, while maintaining compatibility with PHP 5. No features introduced in PHPUnit 5 should be used if they were not present in PHPUnit 4.